source/includes/admonitions/warn-configure-path.rst

```diff
@@ -0,0 +1,20 @@
+.. important::
+
+   To launch your selected MongoDB shell, you must:
+
+   - Install the shell you want to use, and
+
+   - Configure your machine's ``PATH`` environment variable to include 
+     the filepath to your shell binary.
+
+   If you install a new version of your MongoDB shell, update your 
```

Comment:
> If you install a new version of your MongoDB shell, update your ...
=>
If you install a new version of your MongoDB shell, **you will need to** update your ...